.github/workflows/build.yml

```diff
@@ -186,6 +186,32 @@ jobs:
         LATEST_BACKWARD_SUPPORTED_AFM_VERSION: v${{ env.LATEST_BACKWARD_SUPPORTED_VERSION }}
         LATEST_BACKWARD_SUPPORTED_CRD_VERSION: ${{ env.LATEST_BACKWARD_SUPPORTED_VERSION }}
       run: make run-notebook-readflow-bc-tests
+
```

Comment:
do we really need another test job? I seet the TLS tests as a super set of the regular read notebook test. We have 2 read notebook test, with different K8s versions, I'd suggest to use one of them with a regular notebook test and another one with TLS

Reply:
OK, I will change that.